main.py

```diff
@@ -40,9 +39,13 @@ def comments():
 
 
 @app.command()
-def functions_without_docstrings():
+def functions_without_docstrings(directory_path):
     """Determine number of functions without docstrings in a Python directory."""
-    print("In progress")
+    cast_dict = generate_trees(directory_path)
+    functions_dictionary = counter.match_funcdefs(cast_dict)
+    number_missing_docstrings = counter.get_missing_docstrings(functions_dictionary)
+    print(number_missing_docstrings)
```

Comment:
It may be useful to have a better print statement as output for your command:

Even something like: "Number of functions without docstrings: number_here" would be fantastic!!